cypress/e2e/api/customers.spec.cy.js

```diff
@@ -0,0 +1,50 @@
+describe('API Testing Engagesphere', () => {
+  const CUSTOMER_API_URL = `${Cypress.env('API_URL')}/customers`
+
+  it('should return status code 200', () => {
+    cy.request({
+      method: 'GET',
+      url: CUSTOMER_API_URL,
+      failOnStatusCode: false
+    }).then(({ status }) => {
+      expect(status).to.eq(200)
+    })
+  })
+
+  it('should check customer schema', () => {
+    cy.request({
+      method: 'GET',
+      url: CUSTOMER_API_URL,
+      failOnStatusCode: false
+    }).then(({ body }) => {
+      const customers = body.customers
+      expect(customers).to.be.an('array').not.to.be.empty
+
+      const { id, name, employees, size, industry, contactInfo, address } = customers[1]
+      customers.forEach(() => {
+        expect(id).to.be.a('number')
+        expect(name).to.be.a('string')
+        expect(employees).to.be.a('number')
+        expect(size).to.be.a('string')
+        expect(industry).to.be.a('string')
+        expect(contactInfo).to.be.a('object')
```

Comment:
E se `contactInfo` for `null`, visto que esta é uma possibilidade?

Está nas [regras de negócio da API](https://github.com/wlsf82/EngageSphere-Test-Design-Masterclass-Turma-5/blob/main/docs/Requirements.md#response-body), na seção **_notes_**.